plugins/handler/collectd-metrics/main.go

```diff
@@ -79,9 +81,17 @@ func (c *collectdMetricsHandler) Handle(blob []byte, pf bus.MetricPublishFunc) {
 		err = c.writeMetrics(&cdmetric, pf)
 		if err != nil {
 			c.totalDecodeErrors++
+			if reportErrors {
```

Comment:
Great idea! I don't see where reportErrors is defined however. Is this supposed to come in as a configuration? This also needs the bus.EventPublishFunction() to publish the event. This makes me believe we should have one type of handler that gets passed both the EventPublish and MetricPublish function

Reply:
Ah I see what was happening here. The rebase just messed it up

Reply:
Yup, that is rebase error :/ Yeah the bool should be removed now